lib/podio/models/item.rb

```diff
@@ -187,8 +187,11 @@ def find_field_top(field_id, options={:limit => 8})
     end
 
     # @see https://developers.podio.com/doc/items/get-item-count-34819997
-    def item_count(app_id)
-      Podio.connection.get("/item/app/#{app_id}/count/").body
+    # options are either filters {gender: 1, firstname: "Joe"} or a view {view_id: 123}
```

Comment:
Adding option specifics in the code is discouraged. These may be the options now, but not guaranteed in the future. The API docs should be the source of reference.